mock-eye-module/README.md

```diff
@@ -0,0 +1,63 @@
+# Mock Eye Module
```

Comment:
Copyright?

Reply:
We don't have copyrights in the READMEs that I am aware of, but I did forget to add a couple notes here. I'll fix that.